csv2wiki/__init__.py

```diff
@@ -603,31 +615,33 @@ def __init__(self, config, csv_input, null_as_value, msg_out, dry_run_out):
             # There are two kinds of lines:
             #
             #   1) New section indicator (starts with dots)
-            #   2) Column number (starts with a number)
+            #   2) Text line (starts with a pipe)
+            #   3) Comment line (starts with a pound sign)
+            #      - This is not handled explicitly here because
+            #        it's handled by the python config parser
+            #        before we get the sec_map
             #
             # These regexps help us figure out which kind we've got.
             dot_matcher = re.compile("^(\\.+)\\s*(.*)$")
-            col_matcher = re.compile("^(([0-9]+\\s*)+)\\s*.*$")
+            txt_matcher = re.compile("^\\|\\s*(.*)$")
```

Comment:
Aha!  Yes, I was wondering (when I was reading the documentation earlier) if any whitespace immediately following the `|` symbol gets chomped.  It does, which I totally agree is a good thing, but then the documentation for sec_map should say that that happens.

Reply:
This is true for the titles as well, do you want me to update that documentation similarly?